source/index.html

```diff
@@ -1,12 +1,211 @@
+    <header class="main-header">
+
+      <a class="logo" href="#">
+        <img class="logo__logo-image" src="../source/img/logo.svg" width="202" height="auto" alt="Логотип Кэт Энерджи.">
+      </a>
+
+      <nav class="main-nav">
+        <button class="main-nav__toogle" type="button">
+          <span class="visually-hidden">Открыть меню</span>
+        </button>
+        <div class="main-nav__wrapper">
+          <ul class="main-nav__list">
+            <li class="main-nav__item main-nav__item--current">
+              <a class="main-nav__link" href="#">
+                Главная
+              </a>
+            </li>
+            <li class="main-nav__item">
+              <a class="main-nav__link" href="catalog.html">
+                Каталог продукции
+              </a>
+            </li>
+            <li class="main-nav__item">
+              <a class="main-nav__link" href="form.html">
+                Подбор программы
+              </a>
+            </li>
+          </ul>
+        </div>
+      </nav>
+
+    </header>
+
+    <main class="main-container">
+
+      <header class="main-container__header">
+        <h1 class="main-container__title">Функциональное питание для котов</h1>
+        <p class="maiin-container__slogan slogan">
+          Занялся собой? Займись котом!
+        </p>
+        <a class="main-container__button button" href="form.html">Подобрать программу</a>
+      </header>
+
+      <section class="programs">
+        <div class="programs__wrapper">
+          <h2 class="visually-hidden">Программы питания.</h2>
+          <ul class="programs__list">
+            <li class="programs__item">
+              <h3 class="programs__title">Похудение</h3>
+              <p class="programs__description">
+                Ваш кот весит больше собаки и почти утратил способность
+                лазить по деревьям? Пора на диету! Cat Energy Slim поможет
+                вашему питомцу сбросить лишний вес.
+              </p>
+              <a class="programs__link--slim program-slim" href="#">Каталог slim</a>
+            </li>
+            <li class="programs__item">
+              <h3 class="programs__title">набор массы</h3>
+              <p class="programs__description">
+                Заработать авторитет среди дворовых котов и даже собак? Серия Cat Energy Pro поможет вашему коту нарастить необходимые мышцы!
+              </p>
+              <a class="programs__link--pro program-pro" href="#">Каталог pro</a>
+            </li>
+          </ul>
+        </div>
+      </section>
+
+      <section class="product-description">
+        <div class="product-description__wrapper">
+          <h2 class="product-description__title">Как это работает</h2>
+          <ul class="product-description__list">
+            <li class="product-description__item">
+              <p class="product-descriprion__desc product-description__desc--benefit">
```

Comment:
Здесь и дальше опечатки
```suggestion
              <p class="product-description__desc product-description__desc--benefit">
```